src/sass/components/_footer.scss

```diff
@@ -78,3 +78,29 @@ footer {
     }
   }
 }
+
+@media (max-width: 576px) {
+  footer {
+    .footer-menu {
+      text-align: center;
+    }
+
+    .socialmedia ul {
```

Comment:
Czy nie lepiej uzyc bootstrapowej klasy text-center na elemencie z klasa footer-menu oraz na tym ul?

Reply:
Myślałam o tym, ale to spowoduje mi wycentrowanie treści w każdej rozdzielczości, również desktopowej. Chyba, że źle kombinuję? 
![Zrzut ekranu 2019-07-31 o 11 51 36](https://user-images.githubusercontent.com/47738932/62202691-0c9d3980-b38a-11e9-8553-269d95df1750.png)

Reply:
Masz racje, o ile nie ma byc wycentrowane w desktopie :)

Reply:
;) poprawione